pkg/kubelet/dockershim/docker_container.go

```diff
@@ -131,6 +131,10 @@ func (ds *dockerService) CreateContainer(podSandboxID string, config *runtimeapi
 			OpenStdin: config.Stdin,
 			StdinOnce: config.StdinOnce,
 			Tty:       config.Tty,
+			// Disable Docker's health check until we officially support it.
+			Healthcheck: &dockercontainer.HealthConfig{
```

Comment:
But on the other hand, to cherry-pick this into 1.7 and 1.6, we need https://github.com/kubernetes/kubernetes/pull/48308..